pkg/processor/spec_processor.go

```diff
@@ -14,9 +14,9 @@ import (
 )
 
 const (
-	// . is a valid character in a name, cannot use as a separator.
```

Comment:
I don't think this concern was ever valid.  It's a Bundle resource name, not the Kubernetes object name.